sphinx/writers/html5.py

```diff
@@ -277,11 +277,10 @@ def depart_seealso(self, node):
         # type: (nodes.Element) -> None
         self.depart_admonition(node)
 
-    def add_secnumber(self, node):
+    def get_secnumber(self, node):
         # type: (nodes.Element) -> None
```

Comment:
This PR changes the return value of this method. Please update this also.

Reply:
`add_secnumber` is a new method, and `get_secnumber` was not returning anything. I do not understand your point :(

Reply:
@yves-chevallier I would say that tk0miya refers to the type annotation on line 281 which is now wrong because the method does not return None